### ISSUE TITLE
Sync favorites with dropbox

### ISSUE THREAD

**Issue by qnoturas** [jakob/Postico#83]
Similar to 1Password or Jump desktop

**Comment by jakob** [comment]
This sounds like a good idea, but it would require at least a partial rewrite of the favorites system. I currently use Core Data for storing favorites, and Core Data doesn't really play nice with Dropbox. I'd need to switch to storing favorites as files.

Since there are more problems with the current favorite system (see issue #21 about managing large number of favorites), I really have to think about something to improve favorite management. But this is a bit of a bigger task; I'll keep that in mind for a later version of Postico.

**Comment by brandondrew** [comment]
Ugh, I'd like something along these lines but please don't limit/tie it specifically to Dropbox.  I'd prefer to use something else to sync my favorites.

**Comment by qnoturas** [comment]
Absolutely... Be great to dump these favorites out to a json object or some other properties file.

**Comment by jakob** [comment]
Yes, I wouldn't want to tie syncing to to a single provider.

My goal is to store favorites in a folder structure with multiple JSON files. Syncing them could then be done with Dropbox, iCloud, Git, rsync, etc.

I've written some test code, and it looks like it should be feasible, but actually integrating my test code into Postico is going to be a lot of work, so no promises on when this is going to be done.

**Comment by mergulhao** [comment]
👍 for dropbox sync support.
